docs/02-app/02-api-reference/04-functions/unstable_cache.mdx

```diff
@@ -34,15 +34,38 @@ const data = unstable_cache(fetchData, keyParts, options)()
 ```
 
 - `fetchData`: This is an asynchronous function that fetches the data you want to cache. It must be a function that returns a `Promise`.
-- `keyParts`: This is an array that identifies the cached key. It must contain globally unique values that together identify the key of the data being cached. The cache key also includes the arguments passed to the function.
+- `keyParts`: This is an extra array of keys that further adds identification to the cache. By default, `unstable_cache` already uses the arguments and the stringified version of your function as the cache key. It is optional in most cases; the only time you need to use it is when you use external variables without passing them as parameters. However, it is important to add closures used within the function if you do not pass them as parameters.
 - `options`: This is an object that controls how the cache behaves. It can contain the following properties:
-  - `tags`: An array of tags that can be used to control cache invalidation.
+  - `tags`: An array of tags that can be used to control cache invalidation. Next.js will not use this to uniquely identify the function.
   - `revalidate`: The number of seconds after which the cache should be revalidated. Omit or pass `false` to cache indefinitely or until matching `revalidateTag()` or `revalidatePath()` methods are called.
 
 ## Returns
 
 `unstable_cache` returns a function that when invoked, returns a Promise that resolves to the cached data. If the data is not in the cache, the provided function will be invoked, and its result will be cached and returned.
 
+## Example
+
+```jsx
```

Comment:
Let's add a switcher here between `tsx` and `jsx` (like the other snippets in our docs)